lib/ansible/modules/apt.py

```diff
@@ -381,7 +381,7 @@
 from ansible.module_utils.urls import fetch_file
 
 DPKG_OPTIONS = 'force-confdef,force-confold'
-APT_GET_ZERO = "\n0 upgraded, 0 newly installed"
+APT_GET_ZERO = "\n0 upgraded, 0 newly installed, 0 to remove"
```

Comment:
The line we want to match is -

```
"stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\nCalculating upgrade...\nThe following packages will be REMOVED:\n libio-string-perl\n0 upgraded, 0 newly installed, 1 to remove and 0 not upgraded.
```

Previously it was not considering the packages to remove.